keyboards/freyr/freyr.c

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2019 HnahKB
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "freyr.h"
+
+
+void matrix_init_kb(void) {
+  matrix_init_user();
+  led_init_ports();
+};
+
+void matrix_scan_kb(void) {
```

Comment:
This can be removed, since it's not doing anything different.

Reply:
done :D